### PR TITLE
Update to sartools 1.7.3

### PR DESCRIPTION
**Pull request by loraine-gueguen**
Fix issue https://github.com/PF2-pasteur-fr/SARTools-Galaxy/issues/10 and update to sartools 1.7.3

### CODE REVIEW

File: sartools/abims_sartools_edger.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="sartools_edger" name="SARTools edgeR" version="@WRAPPER_VERSION@.1">
+<tool id="sartools_edger" name="SARTools edgeR" version="@WRAPPER_VERSION@.0">
```
Comment (lecorguille) [inline sartools/abims_sartools_edger.xml]: You should adopt the new way to set version `@TOOL_VERSION@+galaxy0`

File: sartools/abims_sartools_deseq2.xml [hunk 2]
```diff
@@ -87,7 +87,7 @@
 	        <param name="adv_param" value="show"/>
```
Comment (lecorguille) [inline sartools/abims_sartools_deseq2.xml]: indentation

File: sartools/macros.xml [hunk 3]
```diff
@@ -100,8 +99,13 @@
...
 
   <macro name="colors_param">
-		<param type="text" size="100" value="dodgerblue,firebrick1,MediumVioletRed,SpringGreen,chartreuse,cyan,darkorchid,darkorange" label="Colors of each biological condition on the plots: 'col1,col2,col3,col4'" argument="--colors" help="Separate the colors with a comma, no space allowed. Default are 'dodgerblue,firebrick1,MediumVioletRed,SpringGreen,chartreuse,cyan,darkorchid,darkorange'." >
+		<param type="text" size="100" value="#f3c300,#875692,#f38400,#a1caf1,#be0032,#c2b280,#848482,#008856,#e68fac,#0067a5" label="Colors of each biological condition on the plots: 'col1,col2,col3,col4'" argument="--colors" help="Separate the colors with a comma, no space allowed. Default are '#f3c300,#875692,#f38400,#a1caf1,#be0032,#c2b280,#848482,#008856,#e68fac,#0067a5'." >
 			<validator type="regex" message="Field requires a value. No space allowed.">\S+</validator>
+                        <sanitizer>
+                                <valid>
+                                        <add value="#" />
+                                </valid>
+                        </sanitizer>
```
Comment (lecorguille) [inline sartools/macros.xml]: You should use space than tab for your indentations

File: sartools/README.rst [hunk 4]
```diff
@@ -8,60 +8,69 @@ Description:
...
+SARTools provides tools to generate descriptive and diagnostic graphs, to run the differential analysis with one of the well known DESeq2 or edgeR packages and to export the results into easily readable tab-delimited files. It also facilitates the generation of a HTML report which displays all the figures produced, explains the statistical methods and gives the results of the differential analysis. Note that SARTools does not intend to replace DESeq2 or edgeR: it simply provides an environment to go with them. For more details about the methodology behind DESeq2 or edgeR, the user should read their documentations and papers.
 
 
 Requirements:
```
Comment (lecorguille) [inline sartools/README.rst]: Not really usefull this section

**Review by lecorguille — CHANGES_REQUESTED** [review]
In the `<token name="@COMMAND*`, for security reason, you have to use some `'` around your inputs, outputs and text param

File: sartools/abims_sartools_edger.xml (hunk 1 above)
Comment (loraine-gueguen) [reply sartools/abims_sartools_edger.xml]: done

File: sartools/README.rst (hunk 4 above)
Comment (loraine-gueguen) [reply sartools/README.rst]: All changes done

File: sartools/abims_sartools_edger.xml (hunk 1 above)
Comment (loraine-gueguen) [reply sartools/abims_sartools_edger.xml]: done

File: sartools/abims_sartools_deseq2.xml (hunk 2 above)
Comment (loraine-gueguen) [reply sartools/abims_sartools_deseq2.xml]: done

File: sartools/macros.xml (hunk 3 above)
Comment (loraine-gueguen) [reply sartools/macros.xml]: done

File: sartools/macros.xml [hunk 5]
```diff
@@ -1,11 +1,10 @@
 <macros>
 
-    <token name="@WRAPPER_VERSION@">1.6.3</token>
+    <token name="@TOOL_VERSION@">1.7.3</token>
 
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="1.6.3">r-sartools</requirement>
-            <requirement type="package" version="1.6.0">r-optparse</requirement>
+            <requirement type="package" version="1.7.3">r-sartools</requirement>
```
Comment (lecorguille) [inline sartools/macros.xml]: You can use the `@TOOL_VERSION@` token here

File: sartools/abims_sartools_deseq2.xml [hunk 6]
```diff
@@ -1,35 +1,35 @@
-<tool id="sartools_deseq2" name="SARTools DESeq2" version="@WRAPPER_VERSION@.1">
+<tool id="sartools_deseq2" name="SARTools DESeq2" version="@TOOL_VERSION@+galaxy0">
     
     <!-- [REQUIRED] Tool description displayed after the tool name -->
```
Comment (lecorguille) [inline sartools/abims_sartools_deseq2.xml]: You can remove those comments

File: sartools/abims_sartools_deseq2.xml [hunk 7]
```diff
@@ -45,26 +45,26 @@
             </param>
             <when value="hide" />
             <when value="show">
-		<expand macro="batch_param" />
```
Comment (lecorguille) [inline sartools/abims_sartools_deseq2.xml:48]: Can you use a section instead of `<conditional name="advanced_parameters" >` ?

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: sartools/macros.xml (hunk 5 above)
Comment (loraine-gueguen) [reply sartools/macros.xml]: done

File: sartools/abims_sartools_deseq2.xml (hunk 7 above)
Comment (loraine-gueguen) [reply sartools/abims_sartools_deseq2.xml:48]: done

File: sartools/abims_sartools_deseq2.xml (hunk 6 above)
Comment (loraine-gueguen) [reply sartools/abims_sartools_deseq2.xml]: done

**Review by lecorguille — APPROVED** [review]

**Pull request merged**